src/mocks/assets/schedule.mock.ts

```diff
@@ -0,0 +1,298 @@
+export const scheduleTestData = [
```

Comment:
Why do we need this file?

Reply:
used it as a mock. theoretically we don't need it anymore